### PR TITLE
card view implemented

### PR DESCRIPTION
**Pull request by iamavishkar**
## JIRA Ticket(s) if any

* [MOB-XXXX](https://iterable.atlassian.net/browse/MOB-XXXX)

## Test Steps

### CODE REVIEW

**Comment by Sagarpatel14** [comment]
image
image

File: src/components/card/index.tsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  onClickSecondaryBtn?: () => void;
+  imgStyle?: CSSProperties;
+  titleStyle?: CSSProperties;
+  cardStyle?: CSSProperties;
+  textStyle?: CSSProperties;
+  primaryBtnStyle?: CSSProperties;
+  secondaryBtnStyle?: CSSProperties;
+}
+
+const Card = (props: CardProps) => {
```
Comment (mprew97) [inline src/components/card/index.tsx:22]: would prefer to keep this consistent with the other components and destructure the props either inline in the component declaration or just below.

File: src/components/card/index.tsx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  const isPrimaryBtnDisabled = props.disablePrimaryBtn || false;
```
Comment (mprew97) [inline src/components/card/index.tsx:23]: these should both just be `Boolean(someProp)`

File: src/components/card/index.tsx [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+            .card {
+              max-width: 90%;
+            }
+          }
+          .card-buttons {
+            display: flex;
+            margin-top: 40px;
+            margin-bottom: 10px;
+            margin-right: 50px;
+          }`}
```
Comment (mprew97) [inline src/components/card/index.tsx:120]: Same comment as the other PR: it's preferred to keep styles out of the component tree as much as possible. if the `style` prop doesn't allow certain css in `style`, we should consider other options.

**Review by mprew97 — CHANGES_REQUESTED** [review]
Please add this along with the other components to a `/components` page in the react-example app.

**Comment by mprew97** [comment]
> request you to please review the PR as it has been updated.

Ty sir!

We should be able to just import the `Card` component in `react-example` as a user would. Can we do that and also have an example of it being displayed on a page somewhere in the example? Like an actual live component example?

**Comment by devcsomnicg** [comment]
Hi @mprew97 here is the video for the same:

https://drive.google.com/file/d/1ifip1OplImM9m2q-kOXGwxNwenqJXkCC/view?usp=sharing

**Comment by mprew97** [comment]
Please resolve the build issues here and then this can be merged.

**Comment by mprew97** [comment]
@devcsomnicg can we resolved the conflict here? happy to merge as soon as it's resolved.

**Pull request merged**